imports/plugins/included/payments-stripe/server/i18n/es.json

```diff
@@ -6,11 +6,11 @@
     "reaction-payments": {
       "admin": {
         "shortcut": {
-          "stripeLabel": "Raya"
+          "stripeLabel": "Stripe"
         },
         "dashboard": {
-          "stripeLabel": "Raya",
-          "stripeDescription": "pagos de la raya"
+          "stripeLabel": "Stripe",
+          "stripeDescription": "pagos de la Stripe"
         },
         "paymentSettings": {
           "stripeLabel": "Raya",
```

Comment:
I don't understand why it fixed the translations 2 lines above but not the translation here